include/struct_defs/struct_0203CDB0.h

```diff
@@ -95,6 +95,6 @@ typedef struct UnkStruct_0203CDB0_t {
     UnkStruct_0209C370 * unk_BC;
     u32 unk_C0;
     UnkStruct_ov66_0222DCE0_sub1 unk_C4;
-} UnkStruct_0203CDB0;
+} FieldSystem;
```

Comment:
Rename file to `include/field_system.h`

Reply:
Or `include/field/field_system.h` (which is probably where it will eventually live).